packages/core/js/Identity.js

```diff
@@ -58,13 +58,13 @@ module.exports = {
    * These IDs are preserved between app upgrades, are saved and restored during the standard application backup process,
    * and are removed at uninstall.
    *
-   * If the current SDK privacy status is \ref AEPMobilePrivacyStatusOptOut, then this operation performs no action.
+   * If the current SDK privacy status is \ref PrivacyStatusOptOut, then this operation performs no action.
    *
    * @param identifiers a dictionary of customer IDs
-   * @param authenticationState a valid \ref AEPMobileVisitorAuthenticationState value.
-   * @see AEPMobilePrivacyStatus
+   * @param authenticationState a valid \ref MobileVisitorAuthenticationState value.
+   * @see PrivacyStatus
    */
-  syncIdentifiersWithAuthState(identifiers?: {string: string}, authenticationState: string) {
+  syncIdentifiersWithAuthState(identifiers?: { string: string }, authenticationState: string) {
```

Comment:
Any reason why `authenticationState` is not of type MobileVisitorAuthenticationState ? Same for the APIs below

Reply:
https://github.com/adobe/react-native-acpcore/blob/05a9b2ed10ccec9e7ae1693913884eaa03c6e060/js/ACPIdentity.js#L67
I just followed what we did in ACPIdentity wrappers. I'm not familiar with Identity extension, please feel free to caret a new PR to fix this issue.

Reply:
In my opinion we can use the enum here for the AEP wrapper. Would you revisit this in the final  review of the Core extensions?

Reply:
Emm ... I think we'd better let our Identity experts fix this issue in another PR.

Reply:
Sounds good, let's revisit this in a separate PR, I created https://github.com/adobe/aepsdk-react-native/issues/96